taskt/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs

```diff
@@ -0,0 +1,533 @@
+                    newFileLink.LinkClicked += NewFileLink_LinkClicked;
+                    flwRecentFiles.Controls.Add(newFileLink);
+                }
+            }
+        }
+
+        private void frmScriptBuilder_Shown(object sender, EventArgs e)
+        {
+            Program.SplashForm.Hide();
+
+            if (_editMode)
+                return;
+
+            AddProject();
+            Notify("Welcome! Press 'Add Command' to get started!");
+        }
+
+        private void pnlControlContainer_Paint(object sender, PaintEventArgs e)
+        {
+            //Rectangle rect = new Rectangle(0, 0, pnlControlContainer.Width, pnlControlContainer.Height);
+            //using (LinearGradientBrush brush = new LinearGradientBrush(rect, Color.White, Color.WhiteSmoke, LinearGradientMode.Vertical))
+            //{
+            //    e.Graphics.FillRectangle(brush, rect);
+            //}
+
+            //Pen steelBluePen = new Pen(Color.SteelBlue, 2);
+            //Pen lightSteelBluePen = new Pen(Color.LightSteelBlue, 1);
+            ////e.Graphics.DrawLine(steelBluePen, 0, 0, pnlControlContainer.Width, 0);
+            //e.Graphics.DrawLine(lightSteelBluePen, 0, 0, pnlControlContainer.Width, 0);
+            //e.Graphics.DrawLine(lightSteelBluePen, 0, pnlControlContainer.Height - 1, pnlControlContainer.Width, pnlControlContainer.Height - 1);
+        }
+
+        private void lblMainLogo_Click(object sender, EventArgs e)
+        {
+            frmAbout aboutForm = new frmAbout();
+            aboutForm.Show();
+        }
+
+        private void frmScriptBuilder_SizeChanged(object sender, EventArgs e)
+        {
+            _selectedTabScriptActions.Columns[2].Width = Width - 340;
+        }
+
+        private void frmScriptBuilder_Resize(object sender, EventArgs e)
+        {
+            //check when minimized
+            if ((WindowState == FormWindowState.Minimized) && (_appSettings.ClientSettings.MinimizeToTray))
+            {
+                _appSettings = new ApplicationSettings().GetOrCreateApplicationSettings();
+                if (_appSettings.ClientSettings.MinimizeToTray)
+                {
+                    notifyTray.Visible = true;
+                    notifyTray.ShowBalloonTip(3000);
+                    ShowInTaskbar = false;
+                }
+            }
+            pnlMain.Invalidate();
+        }
+        #endregion
+        
+        #region Bottom Notification Panel
+        private void tmrNotify_Tick(object sender, EventArgs e)
+        {
+            if (_appSettings ==  null)
+            {
+                return;
+            }
+
+            if ((_notificationExpires < DateTime.Now) && (_isDisplaying))
+            {
+                HideNotification();
+            }
+
+            if ((_appSettings.ClientSettings.AntiIdleWhileOpen) && (DateTime.Now > _lastAntiIdleEvent.AddMinutes(1)))
+            {
+                PerformAntiIdle();
```

Comment:
This seems like an odd thing to do. We just move the mouse every minute? How does idling impact us?

Reply:
If I had to guess...so that your computer doesn't sleep? And a process doesn't crash because of it? I have no idea.